src/shared/log_builder.c

```diff
@@ -36,6 +36,8 @@ static int log_builder_update_hostname(log_builder_t * builder);
  */
 static int log_builder_update_host_ip(log_builder_t * builder);
 
+static int ip_update_interval = 0;
```

Comment:
A short comment will be really helpful to understand this global variable at first sight :+1: